### PR TITLE
add feature to response pdf by file name (resolve #12)

### LINKED ISSUE #12: Response PDF by file path.

### PR DESCRIPTION
**Pull request by drgarcia1986**

### CODE REVIEW

**Comment by drgarcia1986** [comment]
something more Coveralls ? LOL

**Comment by jairhenrique** [comment]
@drgarcia1986 you forgot the documentation :smile:

**Comment by drgarcia1986** [comment]
@jairhenrique done (after one month).
Guys, can you help me with typos and english errors in documentation? ( @jairhenrique and @renanivo )

File: docs/api-guide/responses.md [hunk 1]
```diff
@@ -21,4 +21,22 @@ Arguments:
 
 * `file_name`: The default downloaded file name
 
-You can also include _REST Framework's_ [Response arguments](http://www.django-rest-framework.org/api-guide/responses/#response).
+Another option is _DRF PDF_ `PDFFileResponse`.
+Different from `PDFResponse`, `PDFFileResponse` waiting for the path of pdf file and loads the pdf content from file system.
```
Comment (renanivo) [inline docs/api-guide/responses.md]: I think you could improve this line with something like:

```diff
- `PDFFileResponse` waiting for the path of pdf file...
+ `PDFFileResponse` waits for the path of a pdf file...
```
or

```diff
- `PDFFileResponse` waiting for the path of pdf file...
+ `PDFFileResponse` accepts the path of a pdf file...
```

**Comment by renanivo** [comment]
We could also add headers before the description of `PDFResponse` and `PDFFileResponse` like [DRF's documentation](http://www.django-rest-framework.org/api-guide/responses/) does.

File: docs/api-guide/responses.md (hunk 1 above)
Comment (drgarcia1986) [inline docs/api-guide/responses.md]: Great

**Comment by drgarcia1986** [comment]
@renanivo I think its awesome, but we could make also this on another improvement (with a changelog for example).

**Comment by renanivo** [comment]
Ok :shipit:

**Pull request merged**